angular/src/lib/topbar/topbar.component.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  Component,
+  HostBinding,
+  Input,
+} from '@angular/core';
+
+export type TopbarColor = 'dark' | 'light' | 'blue';
+
+@Component({
+  selector: 'header[cui-top-bar]',
+  template: `
+    <div class="cui-top-bar__container">
+      <div class="cui-top-bar__brand">
+        <ng-content select="brand"></ng-content>
+      </div>
+      <ng-content></ng-content>
+    </div>
+  `,
+  // tslint:disable-next-line:use-host-property-decorator
+  host: {
+    'attr.role': 'navigation',
```

Comment:
Instead of `@HostBinding('class')`, can we use  this to support custom class or ngClass to be passed to his component 
```suggestion
    'attr.role': 'navigation',
    'class': 'cui-top-bar',
    '[class.cui-top-bar--fixed]': 'fixed',
    '[class.cui-top-bar--dark]': 'color === "dark"',
    '[class.cui-top-bar--light]': 'color === "light"',
    '[class.cui-top-bar--blue]': 'color === "blue"'
```

Reply:
updated